### PR TITLE
fix: resolve zod versions to allow AI to pass build

### PR DESCRIPTION
**Pull request by DexterStorey**
## What does this PR do?

The AI app requires a newer version of zod (3.22.2) than the rest of the mono repo (3.20.2)

If the newer version is included in the ai package.json, the other apps will use that package and there will be errors, so we must resolve the older version in the root package and specify the newer version in the ai folder.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by DexterStorey** [comment]
@emrysal I didn't commit the new yarn.lock but I think it might need to be updated to support the new resolutions.

If there is any easy way for me build the lock file locally without the massive diff, please let me know.

**Comment by DexterStorey** [comment]
@baileypumfleet This PR should allow the ai build to pass on vercel

**Review by PeerRich — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -102,7 +102,8 @@
     "@apidevtools/json-schema-ref-parser": "9.0.9",
     "@types/node": "16.9.1",
     "@types/react": "18.0.26",
-    "@types/react-dom": "^18.0.9"
+    "@types/react-dom": "^18.0.9",
+    "zod": "^3.20.2"
```
Comment (PeerRich) [inline package.json:106]: @zomars was this a mistake by me merging?

Comment (zomars) [reply package.json:106]: Zod isn't needed on root AFAIK.

**Pull request merged**